src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -59,13 +55,9 @@ public static void setUp() throws IOException {
     public static void tearDown() {
         // Delete admin user.
```

Comment:
Needs a `assumeTrue(getBooleanEnvVar("RUN_E2E"));` on the first line of this method so that this only runs during E2E testing.

Reply:
Updated.